modin/engines/ray/pandas_on_ray/io.py

```diff
@@ -11,6 +11,8 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""The module holding PandasOnRay factory which performs I/O using pandas on Ray."""
```

Comment:
I think this phrase may confuse a user if we say about original factory PandasOnRay and that you mention.

Reply:
pls elaborate, I don't understand your comment here :)

Reply:
The module holds `PandasOnRayIO` class (not `PandasOnRay` factory) which performs I/O using pandas on Ray. There is an ambiguity between this class ([link](https://github.com/modin-project/modin/blob/master/modin/data_management/factories/factories.py#L173)) and that you mentioned ([link](https://github.com/modin-project/modin/pull/2993/files#diff-dc1f2b528d2a504e16d6b30edf2f043ebea9a8bea40230bc281615c2234eb812R41)) for now.

Reply:
Removed reference to `PandasOnRay` as a term (replaced with `the factory`), this hopefully clears the ambiguity a bit.